src/Moq/Obsolete/MockFactory.cs

```diff
@@ -5,7 +5,7 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
-
+using System.Linq.Expressions;
 using Moq.Properties;
```

Comment:
Please insert a blank line above this line, so that imports are grouped by the main namespace (`System.*`, `Moq.*`).